mb/scripts/mb

```diff
@@ -1000,7 +1000,7 @@ class MirrorDoctor(cmdln.Cmdln):
                         help='set the base directory (so that you can work on a '
                              'subdirectory -- see examples)')
     @cmdln.option('-t', '--target-dir', metavar='PATH',
-                        help='set the target directory (required)')
+                        help='set the target directory (optional, so file hashes can be created as well)')
```

Comment:
This is deprecated. it was announced to be deprecated in 201X already!

Reply:
I still doesn't hurt to have it optional. So e.g. hashes are available outside of DB as well.

Reply:
really ... stuff that has 5+ years of announcement that is deprecated can go.